### DISCUSSION TITLE
Welcome to uiptv Discussions!

### DISCUSSION THREAD

**Discussion by xixogo5105** [xixogo5105/uiptv#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dante6913** [comment]
Hi, thank you for your great work.
It's the first stalker portal player that completely works in linux.
Waiting for epg.
But I'm astonished how this player works in the fist version.
Could we have armhf and arm64 version for the raspberrys?

Thank you

**Reply by xixogo5105** [reply]
on arm/arm64 you can install Java JDK 17, compile and run it. All guidelines are available in readme file.
